context/app/static/js/components/Detail/FileBrowserNode/FileBrowserNode.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import FileBrowserDirectory from '../FileBrowserDirectory';
+import FileBrowserFile from '../FileBrowserFile';
+import { Column } from './style';
+
+function FileBrowserNode(props) {
```

Comment:
Perhaps comment that this is some sort of wrapper around the actual `File` and `Directory` components that one can see.